.github/workflows/distribute-grain.yml

```diff
@@ -0,0 +1,61 @@
+name: Distribute Grain
+on:
+  # A new Cred interval every Sunday. We'll distribute Grain 5 minutes
+  # after the new interval
+  schedule:
+    - cron: 5 0 * * 0 # 00:05 UTC, 16:05 PST
+  push:
+    branches:
+      - "graintest-*" # won't actually update ledger or deploy frontend
+
+jobs:
+  distribute-grain:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "[debug] Check GitHub event name"
+        # This is used to determine whether we actually deploy to GitHub
+        # Pages, so we print it upfront to aid with debugging.
+        run: echo ${{ github.event_name }}
+
+      - name: Install Packages 🔧
+        run: |
+          yarn --frozen-lockfile
+
+      - name: Load Data into Graph 🔌
+        run: |
+          yarn load
+          yarn graph
+        env:
+          SOURCECRED_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SOURCECRED_DISCORD_TOKEN: ${{ secrets.SOURCECRED_DISCORD_TOKEN }}
+
+      - name: Compute Cred 🧮
+        run: |
+          yarn score
+
+      - name: Distribute Grain 💸
+        run: |
+          yarn grain
+
+      - name: Set environment variables
+        run: |
+          echo ::set-env name=PULL_REQUEST_TITLE::"Scheduled grain distribution for week ending $(date +"%B %dth, %Y")"
+          echo ::set-env name=PULL_REQUEST_BODY::"This PR was auto-generated on $(date +%d-%m-%Y) \
+            to add the latest grain distribution to our instance."
+
```

Comment:
If possible--can we save the output of the `yarn grain` command and include that in the PR body?

It prints some info like "distributed XXX grain to YYY identities".

We could even have it output the distribution data in tabular form to make review even easier.